flink-table-store-hive/flink-table-store-hive-connector/src/test/java/org/apache/flink/table/store/hive/TableStoreHiveStorageHandlerITCase.java

```diff
@@ -894,18 +895,18 @@ public void testDateAndTimestamp() throws Exception {
                                 "STORED BY '" + TableStoreHiveStorageHandler.class.getName() + "'",
                                 "LOCATION '" + path + "'")));
         Assert.assertEquals(
-                Collections.singletonList("1971-01-11\t2022-05-17 17:29:20.0"),
```

Comment:
Modify test data to `.1`, because hive3 will produce `17:29:20` for `17:29:20.0`